src/gui/properties/propertieswidget.cpp

```diff
@@ -895,7 +895,7 @@ void PropertiesWidget::editWebSeed()
 void PropertiesWidget::applyPriorities()
 {
     qDebug("Saving files priorities");
-    const QVector<int> priorities = m_propListModel->model()->getFilePriorities();
+    const QVector<BitTorrent::DownloadPriority> priorities = m_propListModel->model()->getFilePriorities();
     // Prioritize the files
     qDebug("prioritize files: %d", priorities[0]);
```

Comment:
https://travis-ci.org/qbittorrent/qBittorrent/jobs/505730099#L1175

Reply:
I would remove this line at all. What do you say?

Reply:
no problem.